pallets/loans/src/tests.rs

```diff
@@ -79,13 +81,20 @@ fn mint_works() {
 }
 
 #[test]
-fn mint_failed() {
+fn mint_must_return_err_when_overflows_occur() {
```

Comment:
And this not related to this PR.

Reply:
I think this is because that I rebased the master